src/Models/EventLocation.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace TruckersMP\APIClient\Models;
+
+class EventLocation
+{
+    /**
+     * The event location.
+     *
+     * @var string
+     */
+    protected $location;
+
+    /**
+     * The event city.
+     *
+     * @var string|null
+     */
+    protected $city;
+
+    /**
+     * Create a new EventLocation instance.
+     *
+     * @param  string  $location
+     * @param  string|null  $city
+     * @return void
+     */
+    public function __construct(
+        string $location,
+        ?string $city
```

Comment:
I made it nullable since, on the event create page, it does not show an asterisk behind the input label:
![image](https://user-images.githubusercontent.com/10339564/141372712-3afa8b53-8d5f-4bad-9ef5-8bf936139d65.png)

However, after testing this, it seems like it just returns "Other" when not choosing a dropdown item. Will refactor this properly, thank you 👍

Reply:
I will pass this feedback on. Thank you for it.